### PR TITLE
[WebGPU] many tests marked are 'Skip' but they pass locally

### PR DESCRIPTION
**Pull request by mwyrzykowski**
#### 8c4e90ef2d55b5ac24b384495af950644e9aa9aa
<pre>
[WebGPU] many tests marked are &apos;Skip&apos; but they pass locally
<a href="https://bugs.webkit.org/show_bug.cgi?id=252842">https://bugs.webkit.org/show_bug.cgi?id=252842</a>
&lt;radar://105840917&gt;

Reviewed by Myles C. Maxfield.

Start enabling tests which are passing locally in order to catch regressions.

* LayoutTests/http/tests/webgpu/webgpu/api/operation/buffers/threading-expected.txt:
* LayoutTests/http/tests/webgpu/webgpu/api/operation/command_buffer/copyBufferToBuffer-expected.txt:
* LayoutTests/http/tests/webgpu/webgpu/api/operation/pipeline/default_layout-expected.txt:
* LayoutTests/http/tests/webgpu/webgpu/api/operation/render_pipeline/vertex_only_render_pipeline-expected.txt:
* LayoutTests/http/tests/webgpu/webgpu/api/operation/texture_view/read-expected.txt:
* LayoutTests/http/tests/webgpu/webgpu/api/operation/texture_view/write-expected.txt:
* LayoutTests/http/tests/webgpu/webgpu/api/operation/uncapturederror-expected.txt:
* LayoutTests/http/tests/webgpu/webgpu/api/validation/createSampler-expected.txt:
* LayoutTests/http/tests/webgpu/webgpu/api/validation/query_set/destroy-expected.txt:
* LayoutTests/http/tests/webgpu/webgpu/idl/constants/flags-expected.txt:
* LayoutTests/http/tests/webgpu/webgpu/idl/exposed.http-expected.txt:
* LayoutTests/http/tests/webgpu/webgpu/idl/exposed.https-expected.txt:
* LayoutTests/http/tests/webgpu/webgpu/web_platform/canvas/getPreferredCanvasFormat-expected.txt:
* LayoutTests/platform/mac/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/261465@main">https://commits.webkit.org/261465@main</a>
</pre>

https://github.com/WebKit/WebKit/commit/80a69bfa51b6c95605d45500355649c597be83de

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/109567 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/18690 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/42318 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/87/builds/1056 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/118700 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/20156 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/9887 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/86/builds/560 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/115323 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/15006 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/98239 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/101836 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/96978 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/29892 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/43213 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/11426 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/31235 "Passed tests") | [❌ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/84985 "Found 2 new API test failures: /WebKitGTK/TestWebKitAccessibility:/webkit/WebKitAccessibility/accessible/event-listener, /WebKitGTK/TestWebKitAccessibility:/webkit/WebKitAccessibility/accessible/state-changed (failure)") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/12085 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/8162 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/17450 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/50842 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/7978 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/13824 "Built successfully") | | | 
| | | | |

### CODE REVIEW

File: LayoutTests/http/tests/webgpu/webgpu/idl/exposed.http-expected.txt [hunk 1]
```diff
@@ -1 +1,5 @@
-(Populate me when we're ready to investigate this test)
+layer at (0,0) size 800x600
+  RenderView at (0,0) size 800x600
+layer at (0,0) size 800x8
+  RenderBlock {HTML} at (0,0) size 800x8
+    RenderBody {BODY} at (8,8) size 784x0
```
Comment (litherum) [inline LayoutTests/http/tests/webgpu/webgpu/idl/exposed.http-expected.txt]: Is this actually passing?

File: LayoutTests/http/tests/webgpu/webgpu/idl/exposed.https-expected.txt [hunk 2]
```diff
@@ -1 +1,5 @@
-(Populate me when we're ready to investigate this test)
+layer at (0,0) size 800x600
+  RenderView at (0,0) size 800x600
+layer at (0,0) size 800x8
+  RenderBlock {HTML} at (0,0) size 800x8
+    RenderBody {BODY} at (8,8) size 784x0
```
Comment (litherum) [inline LayoutTests/http/tests/webgpu/webgpu/idl/exposed.https-expected.txt]: Ditto

File: LayoutTests/platform/mac/TestExpectations [hunk 3]
```diff
@@ -2315,29 +2315,22 @@ imported/w3c/web-platform-tests/xhr/event-timeout-order.any.worker.html [ Failur
 
 webkit.org/b/237166 [ BigSur+ ] webgl/pending/conformance/textures/misc/tex-image-video-repeated.html [ Pass Timeout ]
 
-http/tests/webgpu/webgpu/idl/constants/flags.html [ Skip ]
-http/tests/webgpu/webgpu/idl/exposed.https.html [ Skip ]
-http/tests/webgpu/webgpu/idl/exposed.http.html [ Skip ]
```
Comment (litherum) [inline LayoutTests/platform/mac/TestExpectations]: Can we mark these as [ Pass ] so when we do the next CTS import it's clear which tests we expect to pass vs which tests are new and we have no information about whether they pass or not?

**Review by litherum — APPROVED** [review]
🆒

File: LayoutTests/http/tests/webgpu/webgpu/idl/exposed.http-expected.txt (hunk 1 above)
Comment (litherum) [reply LayoutTests/http/tests/webgpu/webgpu/idl/exposed.http-expected.txt]: Can we figure out why this isn't turning into a text test but is instead a render tree test?

File: LayoutTests/platform/mac/TestExpectations (hunk 3 above)
Comment (mwyrzykowski) [reply LayoutTests/platform/mac/TestExpectations]: Yes good idea

**Pull request merged**